.github/workflows/nightly.yml

```diff
@@ -0,0 +1,48 @@
+name: Build and release nightly APK
+
+on:
+  schedule:
+    # Every nights at 4
+    - cron: "0 4 * * *"
+
+env:
+  CI_GRADLE_ARG_PROPERTIES: >
+    -Porg.gradle.jvmargs=-Xmx4g
+    -Porg.gradle.parallel=false
+    --no-daemon
+
+jobs:
+  nightly:
+    name: Build and publish nightly Gplay APK to Firebase
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Install towncrier
+        run: |
+          python3 -m pip install towncrier
+      - name: Prepare changelog file
+        run: |
+          mv towncrier.toml towncrier.toml.bak
+          sed 's/CHANGES\.md/CHANGES_NIGHTLY\.md/' towncrier.toml.bak > towncrier.toml
```

Comment:
should we commit the _changes_ file after successfully deploying the nightly?

Reply:
I do not think this is necessary, this is just a temporary file which will contain the changelog sent to firebase.